src/libs/sfdk/dockervirtualmachine.cpp

```diff
@@ -491,7 +491,9 @@ QStringList DockerVirtualMachinePrivate::listedImages(const QString &output)
 {
     QStringList images;
     const QStringList lines = output.split(QRegularExpression("[\r\n]"), Qt::SkipEmptyParts);
-    for (const QString &line : lines) {
+    for (QString line : lines) {
+        if (line.startsWith("localhost/"))
+            line = line.right(line.size() - 10);
```

Comment:
This seems like a good workaround to me, but I'm wondering if it would be better to only do it if we detect that docker is really podman? One way to detect could be running "docker --version". Real docker calls itself "Docker", while podman calls itself just "docker".